modules/ossm-integrating-with-user-workload-monitoring.adoc

```diff
@@ -0,0 +1,167 @@
+////
+Module included in the following assemblies:
+* service_mesh/v2x/ossm-observability.adoc
+////
+
+:_content-type: PROCEDURE
+[id="ossm-integrating-with-user-workload-monitoring_{context}"]
+= Integrating with user-workload monitoring
+
+By default, {SMProductName} (OSSM) installs the Service Mesh Control Plane (SMCP) with a dedicated instance of Prometheus for collecting metrics from a mesh. However, production systems need more advanced monitoring systems, like {product-title} monitoring for user-defined projects.
+
+The following steps show how to integrate Service Mesh with User-Workload Monitoring.
+
+.Procedure
+
+. Enable monitoring in user-defined projects by adding the following configuration to the `cluster-monitoring-config` config map:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    enableUserWorkload: true
+----
+
+. Configure the Service Mesh control plane for User-Workload Monitoring:
++
+[source,yaml]
+----
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: basic
+  namespace: istio-system
+spec:
+  addons:
+    prometheus:
+      enabled: false # <1>
+    grafana:
+      enabled: false # <2>
+  meshConfig:
+    extensionProviders:
+    - name: prometheus # <3>
+----
+<1> Disable the default Prometheus instance provided by OSSM.
+<2> Disable Grafana.
+<3> Configure the Prometheus extension provider.
++
+
+. Apply a custom network policy to allow ingress traffic from the `user-workload monitoring` namespace:
++
+[source,yaml]
+----
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: user-workload-access
+  namespace: bookinfo
+spec:
+  ingress:
+  - from:
+    - namespaceSelector:
+        matchLabels:
+          network.openshift.io/policy-group: monitoring
+  podSelector: {}
+  policyTypes:
+  - Ingress
+----
++
+
+. Apply a telemetry object to enable traffic metrics in Istio proxies:
++
+[source,yaml]
+----
+apiVersion: telemetry.istio.io/v1alpha1
+kind: Telemetry
+metadata:
+  name: enable-prometheus-metrics
+  namespace: istio-system # <1>
+spec:
+  selector: # <2>
+    matchLabels:
+      app: bookinfo 
+  metrics:
+  - providers:
+    - name: prometheus
+----
+<1> A telemetry resource created in the control plane namespace applies to all workloads in a mesh. To apply telemetry to only one namespace, create the resource in the target namespace.
+<2> (Optional) Setting `spec.selector.matchLabels` applies the telemetry resource to specific workloads in the target namespace.
++
+
+. Apply a `ServiceMonitor` to monitor the Istio control plane:
++
+[source,yaml]
+----
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: istiod-monitor
+  namespace: istio-system
```

Comment:
```suggestion
  namespace: istio-system # <1>
```

Reply:
@jewertow Do you want "A telemetry resource created in the control plane namespace applies to all workloads in a mesh. To apply telemetry to only one namespace, create the resource in the target namespace." to be repeated here?

Reply:
No, because it's not Telemetry. This `<1>` should point to "<1> Since OpenShift Monitoring ignores the field `spec.namespaceSelector` in `ServiceMonitor` and `PodMonitor` objects, you need to apply the `PodMonitor` in all mesh namespaces, including the control plane namespace." - same as PodMonitor.